api/api_test.go

```diff
@@ -135,9 +136,13 @@ func testConfigAndSetter() (cfg *config.Config, setCfg func(*config.Config) erro
 }
 
 func newTestInstanceWithProfileFromNode(ctx context.Context, node *p2p.QriNode) *lib.Instance {
+	return newTestInstanceWithProfileFromNodeAndOrchestratorOpts(ctx, node, nil)
+}
+
+func newTestInstanceWithProfileFromNodeAndOrchestratorOpts(ctx context.Context, node *p2p.QriNode, o *automation.OrchestratorOptions) *lib.Instance {
```

Comment:
WE NEED TO KILL ALL PLACES THAT CALL THESE. We don't need it any more, now that we can add all these different parameters using `NewInstance` via options.

I already felt like I was adding too much to this pr with the `wftest` package, so I just followed the current patterns to get this to work with `APITestRunner`.

Reply:
It's fine having these as temporary holdovers to make the tests work. Could you please add a TODO in the comment to remove them?